requirements.txt

```diff
@@ -0,0 +1,4 @@
+ply==3.11
```

Comment:
Pinning exact versions has the potential to cause a lot of headaches for downstream users of this library, especially across major Python versions.

Why did you add a requirements file?